docs/sources/copyrights.rst

```diff
@@ -1,47 +1,81 @@
 
-LICENSE:   This software program, its content and documentation are licensed, not sold, to you.  You have a nonexclusive and nontransferable right to use (i.e., to store, load, install, execute and display) the enclosed program in accordance with the documentation.  You may install this program on a single physical system for each license you have purchased.
+1.2 Subject to the terms of this Agreement, 3Laws (a) grants Customer a non-exclusive, non-sublicensable, non-transferable license, solely during the Term, to use the Software solely installed on the robotic system(s) identified in the applicable Order Form solely for Customer’s internal business purposes, and (b) will use commercially reasonable efforts to provide Customer reasonable support services, through electronic mail, chat or another online mechanism, in accordance with 3Laws’ standard practice. 
 
-This program can only be used with a single computer processor located in the United States and any other country to which this software is legally exported; you may physically transfer the program from one computer to another, provided that the program is deleted from the first once its use begins on the second.  This software is considered to be in use with a computer processor when it is loaded into the temporary memory or installed into the permanent memory of that computer, provided that a copy installed on a network server for the sole purpose of distribution to other computers is not "in use."  You agree that the program, its content and its documentation belong to 3Laws Robotics, Inc.  
+1.3 3Laws reserves the right to change or modify portions of this Agreement at any time.  If 3Laws does so, it will post the changes on this page and will indicate at the top of this page the date this Agreement was last revised. 3Laws will also notify Customer, either through the Software user interface, in an email notification or through other reasonable means. Any such changes will become effective no earlier than fourteen (14) days after being posted, except that changes addressing new functions of the Software or changes made for legal reasons may become effective immediately. Customer’s continued use of the Software after the date any such changes become effective constitutes acceptance of the new Agreement.
```

Comment:
We should add the date of last revise at the top of the doc according to this clause